Version 1.1.10 (#9995)
ihhub/fheroes2: changelog.txt

version 1.1.10 (16 July 2025)
- update the Spanish translation
- update the Hungarian translation

> - show the proper message for an empty water chest

Reviewer: ```suggestion
- show the proper message when picking-up an empty water chest
```

- implement the hotkey to toggle the display of the battle turn order
- allow heroes to disembark on proper tiles even if they aren't marked as coast
- remove or move to the nearest water the boats that are placed on land in hacked maps
- update the Russian translation
- fix passability issue for a mountain object
- fix rendering bug for a particular crack object

> - use engine-generated Russian fonts and buttons even if Russian assets are present

Reviewer: ```suggestion
- use engine-generated Russian fonts and buttons even when Russian assets are present
```

- update the German translation
- add a frame to the Swordsman walking animation to make it smoother
- allow to exit Town Settings without applying changes in the Editor
- fix map and players' info update when restarting the map that was changed and saved in the Editor
- fix erasure of object by clicking on its flag in the Editor
- fix placement of an object that overlaps with a flag of an existing object in the Editor
- add a new keyboard icon to the button that opens Virtual Keyboard
- fix the position adjustment logic in case of a tail attack in battles
- fix passabilites update after erasing an object in the Editor
- update the Swedish translation
- remember the position of the game window after closing the application

> - fix scroll cursor ("arrow") position on screen edges for software rendered cursor

Reviewer: ```suggestion
- fix scroll cursor ("arrow") positioning on screen edges for software rendered cursor
```

- update the Hungarian translation
- support 8-directions of attack for wide creatures in battles
- use original button sizes as the minimum size for Editor and Main Menu generated buttons
- add hero casting animation to Teleport, Summon Elemental, Mirror Image, and Earthquake spells animation
- make Editor's File Dialog buttons' size auto-adjustable
- add straight corners to the Magic Book to indicate that a player cannot switch to the next page
- update the Ukrainian translation
- fix white pixels in Sorceress castle sky and rainbow
- add missing bay artwork for Wizard castle
- do not place monsters during the "Month of" event on tiles with Adventure Events
- fix incorrect event removal after removing a nearby object on a map in the Editor
- make random generation code to be platform-independent

> - fix generating buttons font when using non-English assets

Reviewer: ```suggestion
- fix button font generation when using non-English assets
```

version 1.1.10 (20 July 2025)
- limit maximum number of heroes and jails allowed to place in the Editor

> - fix a crash when setting "One side defeats another" victory condition and then removing players from the map

Reviewer: > From what I understood this was when removing all players.

@zenseii, it always happened if you remove all players, but it was not the only case. In example, if you have three players in one alliance and one player in the other and then if you remove only this last player and save the map it has caused a crash.

So we can better say:

- fix a crash when setting victory condition "One side defeats another" and then removing all players that are in one alliance from the map
